### DISCUSSION TITLE
Error Pushing to Git

### DISCUSSION THREAD

**Discussion by springxDxD** [Cyfrin/foundry-full-course-cu#1938] [Q&A]
I'm trying to push my smart contract lottery folder to my github but couldn't. Below is what I did and the result I got. 
```
god@GODs-MacBook-Air foundry-smart-contract-lottery-f23 % git init
Reinitialized existing Git repository in /Users/god/foundry-f23/foundry-smart-contract-lottery-f23/.git/
god@GODs-MacBook-Air foundry-smart-contract-lottery-f23 % git add .
god@GODs-MacBook-Air foundry-smart-contract-lottery-f23 % git add -A
god@GODs-MacBook-Air foundry-smart-contract-lottery-f23 % git status
On branch master
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        modified:   README.md

```
What could be wrong please?

**Comment by EngrPips** [comment]
Hello @springxDxD, you didn't do any push commands in your shared terminal output. Patrick taught step-by-step how to push our code to GitHub in the course `https://updraft.cyfrin.io/courses/foundry/foundry-fund-me/pushing-to-github?lesson_format=video` . Following Patrick's step-by-step guide will really make this easy.

**Reply by springxDxD** [reply]
I didn't yet cause there's nothing to push save the README file. I did "git init", then "git add. ". Running "git status" should display all my files, the ones ready to be pushed and the ones excluded in my gitignore file. But that isn't the case. Kindly check my shared output again and check if I'm doing it wrong

**Reply by EngrPips** [reply]
Hello @springxDxD, Please follow Patrick step by step in this video `https://updraft.cyfrin.io/courses/foundry/foundry-fund-me/pushing-to-github?lesson_format=video` where he teaches how to push your code to Github.
